docs/ref/checks.txt

```diff
@@ -239,6 +239,8 @@ Related Fields
 * **fields.E338**: The intermediary model ``<through model>`` has no field
   ``<field name>``.
 * **fields.E339**: ``<model>.<field name>`` is not a foreign key to ``<model>``.
+* **fields.E340**: Field is using a table that has already been registered
+  by ``<model>``.
```

Comment:
Looking back at other documented I think "Field is using a table that has already been registered by ``<model>/<model>.<field name>``"